test/frontend/components/site/siteBuildLogs.test.js

```diff
@@ -10,13 +10,13 @@ describe('<SiteBuildLogs/>', () => {
   it('should render a download link and <SiteBuildLogTable>', () => {
     const props = {
       params: {
-        buildId: 123,
+        buildId: '123',
       },
       buildLogs: {
         isLoading: false,
         data: [
-          { build: { id: 1 }, output: 'output 1', source: 'source 1' },
-          { build: { id: 1 }, output: 'output 2', source: 'source 2' },
+          { id: 1, output: 'output 1', source: 'source 1', createdAt: '2017-02-11' },
```

Comment:
@jmhooper I missed this previously and we still had a prop validation error in the test logs. Fixed in most recent commit.